model/version/db.go

```diff
@@ -127,6 +128,16 @@ func ByProjectId(projectId string) db.Q {
 		})
 }
 
+// ByProjectIdActivated finds all activated non-patch versions within a project.
+func ByProjectIdActivated(projectId string) db.Q {
+	return db.Query(
+		bson.M{
+			IdentifierKey: projectId,
+			RequesterKey:  evergreen.RepotrackerVersionRequester,
+			ActivatedKey:  true,
+		})
```

Comment:
Is this query indexed?

Reply:
As discussed, this should be changed to a query on the build status